index.html

```diff
@@ -209,6 +209,16 @@ <h3 id="server-performance"><a href="#server-performance">Server performance</a>
             <td class="alert">no</td>
           </tr>
           <tr>
+            <td><a href="https://iojs.org/api/https.html">io.js</a></td>
+            <td class="ok">yes</td>
+            <td class="ok"><a href="https://github.com/joyent/node/pull/5882">yes</a></td>
```

Comment:
I think you could use `SSL_OP_NO_TICKET` to disable tickets, otherwise they will be always sent and accepted. Thus this feature is almost non-configurable, unless doing some crazy stuff :)